plankton/adapters/__init__.py

```diff
@@ -133,6 +138,12 @@ def __init__(self, target_member, property_name):
         self._target_member = target_member
         self._prop = property_name
 
+        try:
+            self.__doc__ = getattr(type(getattr(instance, self._target_member)),
+                                   self._prop).__doc__
+        except AttributeError:
+            pass
```

Comment:
:+1: I only found out while working on the StateMachine, came in handy in the same sort of situation where I wanted to avoid a `try-except` to not catch legitimate errors.